example/BugsnagExample/composeApp/src/androidMain/kotlin/com/example/bugsnag/kmp/MainActivity.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.bugsnag.kmp
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Surface
+import androidx.compose.ui.Modifier
+import com.bugsnag.kmp.Configuration
+
+class MainActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        val config = Configuration(applicationContext, BUGSNAG_API_KEY)
+        startBugsnag(config)
```

Comment:
This config and start should be in an `Application` class rather than in an `Activity`.